src/theme.js

```diff
@@ -5,27 +5,23 @@ export const globalStyles = theme => css`
   // We support five fonts that can be used throughout Mapseed flavors.
   @font-face {
     font-family: "Raleway-Light";
-    src: url("https://fonts.googleapis.com/css?family=Raleway:300&display=swap")
-      format("woff2");
-    font-weight: normal;
+    src: url("https://s3-us-west-2.amazonaws.com/assets.mapseed.org/fonts/Raleway-Light.ttf");
```

Comment:
Note that we're bypassing the CloudFront cache here intentionally, related to https://github.com/jalMogo/mgmt/issues/343.

There might be a better way to manage fonts, but for now this works with the PDF generator.